### ISSUE TITLE
Angular.io for Russia

### ISSUE THREAD

**Issue by naomiblack** [angular/angular.io#1892]
This is a placeholder issue. If you're interested in contributing to the localization project, please comment here and let us know how to reach you.

**Comment by dennis95stumm** [comment]
I want to help. I can also translate to German.
dstumm95@gmail.com

**Comment by eugenyshchelkanov** [comment]
I want to help. eshchelkanov@gmail.com

**Comment by webmaxru** [comment]
@dennis95stumm, @eugenyshchelkanov: please send your full names to salnikov@gmail.com, we started to work on the spreadsheet with contributors contact details.

**Comment by cheerypick** [comment]
As I mentioned earlier, I want to help 
katrineorlova@gmail.com

**Comment by olegkorol** [comment]
If you still need help, oleg@olegkorol.de

**Comment by valorkin** [comment]
We would love to help
Dmitriy Shekhovtsov valorkin@gmail.com
Nikita Demchenko
nikita.demchenko@valor-software.com

**Comment by webmaxru** [comment]
@dennis95stumm, @eugenyshchelkanov, @cheerypick, @olegkorol, @valorkin: спасибо за готовность помочь с локализацией! К активным действиям приступим после AngularConnect

**Comment by webmaxru** [comment]
@dennis95stumm, @eugenyshchelkanov, @cheerypick , @olegkorol, @valorkin: если вы на AngularConnect, то приходите на встречу локализаторов. 28 сентября, 11:20am -12:30pm, Capital Lounge.

To get to the Capital Lounge, go down the escalator to the 2nd floor. As you exit the escalator, turn left and go through the doors. Turn right - our meeting room is the very last one on the right, next to the balcony.

**Comment by webmaxru** [comment]
Начинаем выстраивать инфраструктуру для нашей работы. Разослал вам всем приглашения в Slack-команду локализаторов.

**Comment by Semigradsky** [comment]
I also join
semigradskyd@gmail.com

**Comment by Sprit3Dan** [comment]
If there's still a place, i would love to participate.
sprit3dan@gmail.com

**Comment by webmaxru** [comment]
@Sprit3Dan please send your full name to salnikov@gmail.com

**Comment by webmaxru** [comment]
https://github.com/angular/angular.io/pull/2932/

Good job, @valorkin! But it needs to be checked by the professional proofreader (I see the absence of many commas even in some first lines).

**Comment by valorkin** [comment]
@webmaxru very raw WIP, more interesting approach to translations, and some complicated sentences
BTW it is good job of @omfgnuts :)

**Comment by omfgnuts** [comment]
@webmaxru yup, it's extremely raw, i'm still in the initial translation process, i'm also skipping some sentences for the sake of speed.
